myapp/Gemfile

```diff
@@ -62,6 +64,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'pry-rails'
   gem 'rspec-rails'
+  gem 'bullet'
```

Comment:
to detect N+1 query
ref: https://fablic.qiita.com/craftscat/items/b181b67ddae0c7d0702a